<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationEdit</class>
 <widget class="QDialog" name="AnimationEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>313</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnimationEdit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resourses/images.qrc">
    <normaloff>:/images/Icon16.png</normaloff>:/images/Icon16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,40,0,0,0,10,0,0" columnstretch="40,0,0,0,0,0,0,10">
   <item row="1" column="1" rowspan="6" colspan="3">
    <widget class="QListWidget" name="FramesL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="AddLeft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="title">
      <string>Preview</string>
     </property>
     <widget class="QLabel" name="img">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="pixmap">
       <pixmap resource="../_resourses/images.qrc">:/images/NoAni.png</pixmap>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="playLeft">
     <property name="text">
      <string>Play Left</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="AddRight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="SetRight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="DelLeft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Frames (left)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Frames (right)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Framesets</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QListWidget" name="FramesSets">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="playRight">
     <property name="text">
      <string>Play Right</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Frame delay:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="6" colspan="3">
    <widget class="QListWidget" name="FramesR">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QPushButton" name="DelRight">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="SetLeft">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QSpinBox" name="frameSpeed">
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="6" column="7" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>1000</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../_resourses/images.qrc"/>
 </resources>
 <connections/>
</ui>
